apis/clusters/v1beta1/zookeeper_types.go

```diff
@@ -45,7 +45,8 @@ type ZookeeperStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	ClusterStatus `json:",inline"`
+	ClusterStatus        `json:",inline"`
+	DefaultUserSecretRef *Reference `json:"defaultRef,omitempty"`
```

Comment:
defaultUserRef or userRef

Reply:
It`s not  userRef, so I renamed it to defaultUserSecretRef instead